frontend/src/components/VThemeSelect/VThemeSelect.vue

```diff
@@ -64,7 +64,7 @@ const updateRefs = () => {
 }
 
 onMounted(updateRefs)
-watch(() => [darkMode.colorMode.value, darkMode.osColorMode.value], updateRefs)
+watch([darkMode.colorMode, darkMode.osColorMode], () => updateRefs())
```

Comment:
How is `() => updateRefs()` different from `updateRefs`? Does it fix a problem or is it a stylistic/conventional thing?

Reply:
I'm not certain. I added this change trying to fix the cases where Playwright would select an option, but the selected option was still the same on the next line. I think the previous way has one more layer of indirection than what is here, and maybe the change happens in a different tick of the event loop so that Playwright sees the not-yet-changed version?